internal/tui/tui.go

```diff
@@ -65,7 +63,8 @@ func (m *model) Init() tea.Cmd {
 			return errorMsg{err}
 		}
 		if enabled {
-			return radioEnabledMsg{}
+			// This will be handled by the list view's OnEnter
+			return nil
```

Comment:
is this logic correct? should we be returning early if wireless is enabled? shouldn't it be inverted?